packages/@o3r/design/builders/generate-css/schema.json

```diff
@@ -33,7 +33,7 @@
     },
     "rootPath": {
       "type": "string",
-      "description": "Root path of files where the CSS will be generated"
+      "description": "Root path used to determine the CSS file to edit when specified by the token extension (will use the file where the Design Token is defined as `rootPath` if not specified)"
```

Comment:
When `$extensions.o3rTargetFile` is specified in the Design Token it indicate a file where to generate the CSS Var.
It can not really be an absolute path, so it is a relative path.
Per default it is relative to the Design Token.json path but, if specified, it can be relatif to the `rootPath`.

Reply:
```suggestion
      "description": "Root path used to determine the CSS file to edit if specified by the o3rTargetFile token extension. It will default to the Design Token file folder."
```
I don't know if this is clearer?